db/models.py

```diff
@@ -1,4 +1,12 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 from django.db import models
+from django.db.models import UniqueConstraint
+
+
+class User(AbstractUser):
+    pass
```

Comment:
The `User` class is defined but not used in the code. Consider using `get_user_model()` as recommended in the checklist: 'Use `get_user_model()` instead of `User`, it is the best practice.'